lib/dialects/postgres/connection-manager.js

```diff
@@ -186,6 +186,10 @@ class ConnectionManager extends AbstractConnectionManager {
         }
       }
 
+      if (this.sequelize.config.intervalStyle) {
+        query += `SET intervalstyle TO ${this.sequelize.config.intervalStyle};`;
```

Comment:
Postgres returns intervals as strings. But we need ISO format to be able to accurately parse them.
See https://www.postgresql.org/docs/current/datatype-datetime.html#DATATYPE-INTERVAL-OUTPUT